lib/src/view/design_main_loopout_length.dart

```diff
@@ -0,0 +1,57 @@
+import 'dart:html';
+import 'dart:math';
+
+import 'package:over_react/over_react.dart';
+import 'package:built_collection/built_collection.dart';
+import 'package:platform_detect/platform_detect.dart';
+import 'package:scadnano/src/state/group.dart';
+import 'package:scadnano/src/view/design_main_strand_loopout.dart';
+import 'package:scadnano/src/view/transform_by_helix_group.dart';
+import 'package:tuple/tuple.dart';
+
+import '../state/helix.dart';
+import 'package:scadnano/src/state/geometry.dart';
+import '../state/strand.dart';
+import '../state/domain.dart';
+import '../state/loopout.dart';
+import 'pure_component.dart';
+import '../util.dart' as util;
+
+part 'design_main_loopout_length.over_react.g.dart';
+
+UiFactory<DesignMainLoopoutProps> DesignMainLoopout = _$DesignMainLoopout;
+
+mixin DesignMainLoopoutPropsMixin on UiProps {
+  Geometry geometry;
+  Loopout loopout;
+}
+
+class DesignMainLoopoutProps = UiProps with DesignMainLoopoutPropsMixin;
+
+class DesignMainLoopoutComponent extends UiComponent2<DesignMainLoopoutProps> with PureComponent {
+  @override
+  render() {
+    List<ReactElement> loopout_length_elts = [];
+    loopout_length_elts.add(this._loopout_length(props.loopout));
+    return (Dom.g()..className = 'strand-loopout-length')(loopout_length_elts);
+  }
+
+  ReactElement _loopout_length(Loopout loopout) {
+    var start_offset = '50%';
+    var dy = '${0.1 * props.geometry.base_width_svg}';
+
+    Map<String, dynamic> style_map = {'fontSize': '${9}px'};
+    var loopout_length = loopout.loopout_length.toString().split('').reversed.join('');
+
+    SvgProps text_path_props = (Dom.textPath()
+      ..className = 'dna-seq-loopout'
```

Comment:
`'dna-seq-loopout'` doesn't seem like a good className for this element. It represents a loopout length, so I'd call it `loopout-length` instead.

Also, I'm now starting to put these strings as constants in constants.dart that begin with `'css_selector_'`:

https://github.com/UC-Davis-molecular-computing/scadnano/blob/dc1e16f3209f71239fff5a672b2b5101f4426927/lib/src/constants.dart#L315-L340

This is especially important for anything that will be referenced in the file [scadnano-styles.css](https://github.com/UC-Davis-molecular-computing/scadnano/blob/master/web/scadnano-styles.css).

In fact, if possible I'd like to figure out a way to have a single place to define these constants that can be referenced from both Dart and from the css file, but I don't know how to do that. But collecting all of the Dart CSS selector strings into one place is helpful.

Reply:
Note that this is not currently followed uniformly in the source code, but I'm trying to update it whenever I see css selector classes that are not defined by a constant in constants.dart.